base.mako

```diff
@@ -68,5 +68,35 @@
             ga('create', 'UA-104682927-1', 'auto');
             ga('send', 'pageview');
         </script>
+        <script>
+(function () {
+  function copyTextToClipboard(text) {
+    var textArea = document.createElement("textarea");
+    textArea.value = text;
+    textArea.style.top = "0";
+    textArea.style.left = "0";
+    textArea.style.position = "fixed";
+    document.body.appendChild(textArea);
+    textArea.focus();
+    textArea.select();
+    document.execCommand('copy');
+    document.body.removeChild(textArea);
+  }
+  var codeBlockElements = document.querySelectorAll('div#advanced ul')[8].getElementsByClassName("highlight");
+  for (let block of codeBlockElements) {
```

Comment:
I believe `for (let ... of ...) {` is less portable -- caniuse says only 93%: https://caniuse.com/#feat=mdn-javascript_statements_for_of

Reply:
I updated the script to use the normal `for(var i=0; ....; i++)`